packages/blade/src/components/StepGroup/StepLine.web.tsx

```diff
@@ -16,6 +15,25 @@ import BaseBox from '~components/Box/BaseBox';
 import type { StyledPropsBlade } from '~components/Box/styledProps';
 import Svg, { Path } from '~components/Icons/_Svg';
 import { makeSize, useTheme } from '~utils';
+import { throwBladeError } from '~utils/logger';
+
+// universal base64 encode
+const encode = (svgString: string): string => {
+  try {
+    if (typeof Buffer !== 'undefined') {
+      return Buffer.from(svgString).toString('base64');
+    }
+
+    return window.btoa(svgString);
+  } catch (error: unknown) {
+    throwBladeError({
+      message: `Failed to encode SVG string: ${error}`,
+      moduleName: 'StepLine',
+    });
+
+    return '';
+  }
+};
```

Comment:
can we the original code and attribute the author ? 
 https://github.com/blakeembrey/universal-base64/blob/master/src/browser.ts
 
 this code looks fine ... but might have some edge cases

Reply:
we can skip this as well.. since it's used for only one

Reply:
So I checked for the edge cases. `window.btoa` fails on emojis and characters outside of latin1.

Since we're using this on static string defined by us, its fine to not cover those edge cases. I tried and it works on our strings. 

Anyways if we're using this on some dynamic string later, we'll need a sanitise function to avoid XSS and plain encoding won't work